bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/ColumnDescriptorAdapter.java

```diff
@@ -194,8 +194,9 @@ public static String buildGarbageCollectionExpression(HColumnDescriptor columnDe
   private static void convertGarbageCollectionExpression(String gcExpression,
       HColumnDescriptor columnDescriptor) {
     if (Strings.isNullOrEmpty(gcExpression)) {
-      // No GC Expression means keep all versions, MaxVersions = 0.
-      columnDescriptor.setMaxVersions(0);
+      // No GC Expression means keep all versions. HBase doesn't support this, but this is the
+      // default Bigtable behavior. 
```

Comment:
Discussed with Marcal. I understand what this is now.
- There are two type of adapters; one translates from HBase to Bigtable and one translates from Bigtable  to HBase; mixing them in the same package/file confused me.

- The change here basically makes the correctly translation from Bigtable (no gc set) to HBase (all versions). We need to make sure the reverse is correct: HBase (all versions) -> Bigtable (no gc set).

- a unit test that test the translation is reversible would be nice.